src/align_oc_msg_colons.cpp

```diff
@@ -124,10 +124,10 @@ void align_oc_msg_colon(chunk_t *so)
    {
       chunk_t chunk;
 
-      chunk.type        = CT_SPACE;
+      set_chunk_type(&chunk, CT_SPACE);                    // Issue #2567
```

Comment:
Also, all this `&chunk` stuff is making me wonder if an overload that takes a `chunk_t&` would be useful? :wink:

If we're feeling *really* clever, I wonder if it would be interesting to change all `chunk_t*` parameters to:
```c++
template <typename T>
class auto_ptr_from_ref
{
public:
  auto_ptr_from_ref(T* p) : m_ptr{p} {}
  auto_ptr_from_ref(T* p) : m_ptr{p} {}
  auto_ptr_from_ref(T& r) : m_ptr{&r} {}
  operator T*() const { return m_ptr; }
  T* operator->() const { return m_ptr; }
  T& operator*() const { return *m_ptr; }
private:
  T* m_ptr;
};
using chunk_ptr = auto_ptr_from_ref<chunk_t>;

Reply:
Let us make it with another PR.
I am working on #2281, which need some changes such as here.
Some other are coming soon.